external-docs/docs/glossary/index.md

```diff
@@ -1,3 +1,70 @@
 # Glossary
 
-todo: Write something here. Issue #259.
+### Aggregate
+An aggregate is formed when multiple numbers are gathered for statistical purposes and are expressed as one number. This could be in the form of a total or an average<sup>[[10]](https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Beginners:Statistical_concept_-_Aggregate)</sup>.
+
+### Attribute
+The attribute components allow us to qualify and interpret the observed value(s). They 	enable specification of the units of measure, any scaling factors and metadata such as the 	status of the observation (e.g. estimated, provisional)<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Code List
+Predefined set of terms from which some statistical coded concepts take their values<sup>[[4]](https://sdmx.org/wp-content/uploads/SDMX_Glossary_version_2_1-Final-2.docx)</sup>.\
+Example Code List:
+| Label            | Notation       | Parent Notation | Sort Priority |
+|------------------|----------------|-----------------|---------------|
+| Wine             | wine           |                 | 1             |
+| Made-Wine        | made-wine      |                 | 2             |
+| Spirits          | spirits        |                 | 3             |
+| Beer and Cider   | beer-and-cider |                 | 4             |
+| Beer             | beer           |                 | 5             |
+
+### CSV-W
+Comma Separated Values on the Web – a standardised format to express useful metadata about CSV files and other kinds of tabular data. For more information, see the W3C project page on this topic<sup>[[7]](https://www.w3.org/TR/tabular-data-primer/)</sup>.
+
+### Dimension
+The dimension components serve to identify the observations. A set of values for all the dimension components is sufficient to identify a single observation. Examples of dimensions include the time to which the observation applies, or a geographic region which the observation covers<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Literal
+Basic values such as strings, dates, booleans, and numbers that can only be used in the object position of an RDF triple. To allow correct parsing and interpretation, literals are best associated with a datatype<sup>[[8]](http://www.proxml.be/losd/semcubes.html)</sup>.
+
+ ### Measure
+The measure components represent the phenomenon being observed<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
+
+### Observation
+A data point which can be characterised by a dimension that defines what the data point applies to (e.g. time, area, gender) along with metadata describing what has been measured (e.g. economic activity, population), how it was measured and how the data point is expressed (e.g. units, multipliers, status)<sup>[[1]](https://www.w3.org/TR/vocab-data-cube/#cubes-model)</sup>.
```

Comment:
I'd say something more along the lines of:

`An observation is the RDF Data Cube representation of an observed value. It describes what has been measured, its value, the units in which it has been measured as well as the dimension values necessary to identify the subset of the population which has been measured.`